serverless_llm/cli/deploy.py

```diff
@@ -72,6 +72,22 @@ def __init__(self, args: Namespace) -> None:
             os.path.dirname(__file__), "default_config.json"
         )
 
+        self.validate_args()
+
+    def validate_args(self) -> None:
```

Comment:
Call `validate config`. All requirements must be applied in one place to avoid potential inconsistency.